plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -74,6 +74,30 @@ export const DEFAULT_OBJECTS: ObjectToFetch[] = [
     plural: 'ingresses',
     objectType: 'ingresses',
   },
+  {
+    group: 'batch',
+    apiVersion: 'v1',
```

Comment:
we are currently on an older version of k8s (1.18) and still using `v1beta1` apiVersion.

Would it be possible to add both? I assume it will only show the ones it's able to retrieve? similar to what happens for ingresses currently on older versions of k8s